### PR TITLE
Set col width to auto

### PR DESCRIPTION
**Pull request by libremente**
This PR sets a col width to auto in order to obtain a better rendering of the swagger specs. 
Schermata 2019-08-17 alle 13 12 25

Fixing #359, third point.
@sebbalex `main.scss` might not be the best place where to store such a rule, should we move it somewhere else?

### CODE REVIEW

**Comment by sebbalex** [comment]
I'd suggest to use the content specific column class to get the ideal width, which would be:

```css
td.col.response-col_status {
    width: auto;
}
```
and move the rule in specific swagger style area `swagger/components/CustomLayout.css`
Please check my last commit on this.

**Comment by libremente** [comment]
I had already tried to insert such a role in the `customLayout.css`, however I don't see the change reflected in the swagger page. Can you see it? @sebbalex 
Schermata 2019-08-19 alle 14 25 46

File: swagger/components/CustomLayout.css [hunk 1]
```diff
@@ -27,4 +27,8 @@
   .swagger--head-card {
     top: 0;
   }
+}
+
+td.col.response-col_status {
```
Comment (sebbalex) [inline swagger/components/CustomLayout.css]: Some areas in request were not matched by this rule. We should change it to:

```css
tr > *.col {
    width: auto;
}
``` 
@libremente

Comment (libremente) [reply swagger/components/CustomLayout.css]: Ok, let me push that and squash @sebbalex 👍

**Pull request merged**